pkg/bazel/bazel.go

```diff
@@ -94,3 +95,31 @@ func (b *bazel) Flags() (map[string]*FlagInfo, error) {
 
 	return flags, nil
 }
+
+func (b *bazel) AQuery(query string) (*ActionGraphContainer, error) {
+	r, w := io.Pipe()
+	agc := &ActionGraphContainer{}
+
+	bazelErrs := make(chan error, 1)
+	defer close(bazelErrs)
+	go func() {
+		defer w.Close()
+		_, err := b.RunCommand([]string{"aquery", "--output=proto", query}, w)
+		bazelErrs <- err
+	}()
+
+	protoBytes, err := ioutil.ReadAll(r)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get Bazel aquery output: %w", err)
+	}
+
+	if err := <-bazelErrs; err != nil {
+		return nil, fmt.Errorf("failed to run Bazel aquery: %w", err)
+	}
+
+	proto.Unmarshal(protoBytes, agc)
+	if err := proto.Unmarshal(protoBytes, agc); err != nil {
+		return nil, fmt.Errorf("Failed to parse ActionGraphContainer: %w", err)
```

Comment:
In this case, it's okay to add some extra information to the error, and it won't break the Go 2 error handling.

```suggestion
		return nil, fmt.Errorf("failed to run Bazel aquery: parsing ActionGraphContainer: %w", err)
```